compose/config.py

```diff
@@ -330,6 +333,17 @@ def resolve_host_path(volume, working_dir):
         return container_path
 
 
+def resolve_build_path(build_path, working_dir=None):
+    if working_dir is None:
+        raise Exception("No working_dir passed to resolve_build_path")
```

Comment:
if `working_dir` is required it really shouldn't be a `kwarg`, it should  be just a regular positional argument